### ISSUE TITLE
A11y_SQLdatabase projects extension_Advance Publish_Exclude Object Type_Voiceover : Voiceover is announcing incorrect state as unticked for selected tick box under "Exclude Object Type" table.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/azuredatastudio#22187]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #MAC; #AINotInScope; #WCAG1.3.1; #Info and Relationships; #Element:checkbox; #Voiceover;

### Actual Result:​
 Voiceover is announcing incorrect state as unticked for selected tick box under "Exclude Object Type" table.
### Expected Result:​
 Voiceover should announce the correct state as ticked for selected tick box under "Exclude Object Type" table and unticked for non-selected tick box.

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps:

1.Start Voiceover with using shortcut key Command+F5.
2.Launch Azure Data Studio
3.Tab till extension and search for “SQL Database Projects” and install it.
4.Tab till database project and press enter.
5.“Database Project” panel will open.
6.Tab till existing project name and press context menu button(right click).
7.Tab till "Publish" menu-item in list and press enter.
8.Tab till "Advance" button and press enter.
9.Tab till "Exclude Object Type" tab and press enter.
10.Tab till any cell and press enter to tick the tick-box.
11.Verify whether Voiceover is announcing incorrect state as unticked for selected tick box under "Exclude Object Type" table or not.

### User Impact:
Voiceover user will get incorrect information about the current state of the record as after selecting also voiceover is announcing it as unselected.

### Attachment:

![advance publish_ tick box](https://user-images.githubusercontent.com/91586101/223466049-1d9e1658-df46-442c-b5ee-eefb2b6144fe.gif)

**Comment by kisantia** [comment]
@manaswita-chougule this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!
